plugins/grpc/swift-protobuf/v1.0.0/buf.plugin.yaml

```diff
@@ -0,0 +1,23 @@
+version: v1
+name: buf.build/grpc/swift-protobuf
+plugin_version: v1.0.0
+source_url: https://github.com/grpc/grpc-swift-protobuf
+description: Generates Swift client and server stubs for the gRPC framework.
+deps:
+  # https://github.com/grpc/grpc-swift-protobuf/blob/1.0.0/Package.swift#L42
+  - plugin: buf.build/apple/swift:v1.28.1
```

Comment:
Should we update this to the latest 1.28.2?

Reply:
Do we try to normally match the latest patch release? I dropped it down in https://github.com/bufbuild/plugins/pull/1723/commits/c6e380cb6f6ff50a05d3101396d5e5e66dec51b7 to match the dep in Package.swift; happy to match the latest though.

Reply:
As long as it is compatible we typically bump the deps to the latest version.

Reply:
https://github.com/bufbuild/plugins/pull/1723/commits/bf7eb42435b5eb78c8bc4b1e0c9536d5cc78e7d9, thanks!